sort_function.py

```
@@ -0,0 +1,35 @@
+
+
+def choose_cover(files: list[str]) -> str:
+    if not files:
+        raise ValueError("Empty file list")
+
+    ranked: list[tuple[int, int, str]] = sorted((score(f) for f in files))
+    return ranked[0][-1]
+
+
+def sort_by_cover_likelihood(files: list[str]) -> list[str]:
+    return [t[-1] for t in sorted((score(f) for f in files))]
```

Ruff (0.11.9)

Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)